components/admin_console/admin_definition.jsx

```diff
@@ -3270,7 +3293,7 @@ const AdminDefinition = {
                         success_message: t('admin.ldap.testSuccess'),
                         success_message_default: 'AD/LDAP Test Successful',
                         isDisabled: it.any(
-                            it.not(it.userHasReadPermissionOnResource('authentication')),
+                            it.not(it.userHasReadPermissionOnResource(RESOURCE_KEYS.AUTHENTICATION.LDAP)),
```

Comment:
This should be a Write Permission and not a read as Max pointed out in the Server PR.

Reply:
Give the ancillary of this endpoint to WRITE_AUTH instead of READ_AUTH on server side